BizHawk.Client.EmuHawk/MainForm.cs

```diff
@@ -480,6 +480,19 @@ public int ProgramRunLoop()
 					GlobalWin.Tools.LuaConsole.ResumeScripts(false);
 				}
 
+				if (GlobalWin.Tools.Has<TAStudio>())
+				{
+					// There are instances where the movie will be switched out of recording mode but TAStudio doesn't know about it

```

Comment:
Reproduced, and it seems it's emuhawk who doesn't know it's disabled read|write, because look at the movie mode icon in the mainform. It seems it doesn't know it's not recording anymore if we're appending frames out of order, as in, at the current frame and after. So instead of fixing just tastudio, we need emuhawk to know to update itself.

Reply:
That can be fixed with a `SetMainformMovieInfo();`
But, does this situation occur anywhere outside of TAStudio? If not I can just throw that in after the checkbox update.

If it does, well I guess I don't even know where to look to reproduce it.